read-the-docs-2018-stats.rst

```diff
@@ -0,0 +1,107 @@
+.. post:: Jan 7, 2019
+   :tags: stats, year-in-review
+
+Read the Docs 2018 Stats
+========================
+
+2018 was another good year for Read the Docs.
+We've settled into a sustainability model that is working for us,
+and have a team of 5 folks working full-time on the project.
+
+Here are our stats for the past year,
+which we've published for the past 6 years.
+This is part of our effort to be transparent in our organization,
+as well as our source code.
+
+.. note:: 
+
+	You can always see our stats for the last `30 days`_. 
+
+	Our posts from 2013_, 2014_, 2015_, 2016_, and 2017_ are also available.
+
+.. _30 days: http://www.seethestats.com/site/readthedocs.org
+.. _2013: https://blog.readthedocs.com/read-the-docs-2013-stats/
+.. _2014: https://blog.readthedocs.com/read-the-docs-2014-stats/
+.. _2015: https://blog.readthedocs.com/read-the-docs-2015-stats/
+.. _2016: https://blog.readthedocs.com/read-the-docs-2016-stats/
+.. _2017: https://blog.readthedocs.com/read-the-docs-2017-stats/
+
+Page Views
+----------
+
+Our stats:
+
+* 400 Million Page Views (up from 338M)
+* 77 Million Unique Visitors (up from 75M)
+
+.. From Google Analytics
+
+These numbers are not as accurate this year as in the past,
+because we have :doc:`implemented Do Not Track support </do-not-track>` to our Google Analytics setup.
+So we're losing a non-trivial number of users in that 
```

Comment:
A missing period at the end of this sentence

Reply:
I would actually get a little more specific and say something to the affect of "since we implemented Do Not Track in May, our analytics pageview figures understate our actual traffic".